README.md

```diff
@@ -1,54 +1,129 @@
-# Technology
+
+# ChallengeGo
+
+O ChallengeGo é uma API para um simples sistema bancário, com os seguintes objetivos:
+
+* Criar uma conta;
+* Autenticar-se;
+* Realizar depósitos;
+* Realizar transferências;
+* Realizar saques;
+
+A API acessa um banco de dados em MySQL, onde são mantidas duas tabelas:
+* account
+* transfer
+
+O banco é versionado com migrate. Ao fazer o deploy da API, as tabelas serão criadas. Para alterar as tabelas, versionando o banco, podem ser executados os comandos:
+
+
+```bash
+
+migrate -path gateway/db/mysql/migration -database "mysql://root:password@tcp(localhost:3306)/sys?multiStatements=true" -verbose up
+
+```
+
+<h2>Rotas da aplicação</h2>
+
+A API está documentada com swagger que se encontra na url http://localhost:8080/docs/swagger/
```

Comment:
hey @Francisleide ! I'd mention this url after telling how to run, maybe this section could be after "Como utilizar"